kinto/core/resource/__init__.py

```diff
@@ -1095,7 +1095,7 @@ def _extract_filters(self):
                         send_alert(self.request, message, url)
                     operator = COMPARISON.LT
 
-                if value == "":
+                if value == "" or isinstance(value, dict):
```

Comment:
There might other values that we don't support here (eg. `[]`). So it might be safer to check for the supported types instead (string or number).

Reply:
@leplatrem Updated the PR. please review.